baldur/README.md

```diff
@@ -22,33 +22,38 @@ TODO: add support for allowedGroup to give service access.
 org.opencadc.baldur.allowedGroup = {groupURI}
 
 # one or more entry properties to grant permission to access artifacts
-# - each entry has a name and an Artifact URI pattern (regex)
+# - each entry has a name
+# - an Artifact URI pattern (regex)
```

Comment:
this comment still says "space separated list of" which is not correct: "zero or more group permissions" would be more accurate